docs/src/SurfaceFluxes.md

```diff
@@ -141,3 +141,79 @@ u(z) = u_{\star} \frac{F_m(z,z_{0m}, L_O)}{\kappa} + u_{sfc},
+
+\subsubsection{1. MOST in RSL and exponential canopy profiles}
+Evidently, MOST could be used in the RSL with the origin of the profile would be $z=d$ (instead of $z=0$ as in MOST), as was suggested in \cite{WangCionco2007}. However, this would give a zero wind at $d$, which is rarely the case in real canopies: 
+```math
+\begin{equation}
+    \frac{\partial u (z)}{\partial z} = (u_{\star}/\kappa (z-d)) \Phi_M(\zeta), 
+\end{equation}
+```
+with similar form suggested for scalars, except with a much smaller roughness length, to account for the "bluff body" effect (canopy is much more efficient at absorbing momentum compared to its ability to emit/absorb heat). 
+
+For the layer within the deep canopy, $z < d$, an exponential profile is suggested for the wind profile:
+```math
+\begin{equation}
+u(z) = u_h \exp((z/h_c - 1) \alpha)
+\end{equation}
+```
+
+where $u_h = u(h_c)$, and $\alpha$ is the canopy flow index, dependent on the leaf morphology, density, element flexibility, geometry and (in some cases) the wind speed. Typically this is approximated by a constant, depending on the land use category. It is unclear how the authors avoid discontinuities at $h_c$. Note, the authors also suggest a linear interpolation-based solution around the forest edges, which is dependent on the wind direction. 
+
+Note that CLM4.5 also uses MOST, and its canopy layer ignores the direct effects of turbulence, parameterizing $u$ as $u_*$. This is suboptimal. See \cite{Bonan2018} for more details. (CLM4.5 uses a parameterisation that assumes that the wind speed within the canopy is equal to the friction velocity $u_{\star}$.)
+
+#### Modified MOST in RSL
+Following \cite{Physick1995}, we can provide extensions to the MOST to capture the flow behaviour within the roughness-sublayer whose maximum height is denoted by $z_{*}$. Equation (2) in \cite{Physick1995} corresponds to (\ref{eq:F_h_fin-diff}) in this document, and gives the velocity profiles in the surface layer following the canonical MOST. For model levels $z$ such that $d \leq z \leq z_{\star}$ (within the RSL),  \cite{Physick1995} suggest the following modification to the velocity gradient profile. The profile would be:
+
+```math 
+\begin{equation}
+    \frac{\partial u (z-d)}{\partial z} = (u_{\star}/\kappa z) \Phi_M(\zeta)\phi_M(z-d/z_{\star}-d), 
+\end{equation}
+```
+
+where $\Phi_M$ is the stability function applied in the canonical MOST theory, 
+```math
+\begin{equation}
+    \phi_M\Big(\frac{z-d}{z_{\star}-d}\Big) = 0.5\exp(\ln(2)\frac{z-d}{z_{\star}-d}),
+\end{equation}
+```
+and is estimated from flux-tower data over flat, tree-covered terrain with $z_{0}$ ranging from $0.4-0.9 ~\mathrm{m}$. The stability parameter is defined by $\zeta = (z-d)/L$ following the zero-plane displacement correction.
```

Comment:
Does this refer to roughness length? I thought it can assume a much wider range of values.